BlockSettleUILib/AuthAddressConfirmDialog.cpp

```diff
@@ -102,12 +102,36 @@ void AuthAddressConfirmDialog::onError(const QString &errorText)
    reject();
 }
 
-void AuthAddressConfirmDialog::onAuthAddressSubmitError(const QString &address, const QString &error)
+void AuthAddressConfirmDialog::onAuthAddressSubmitError(const QString &address, const bs::error::AuthAddressSubmitResult statusCode)
 {
    progressTimer_.stop();
+
+   QString errorText;
+
+   switch (statusCode) {
+   case bs::error::AuthAddressSubmitResult::SubmitLimitExceeded:
+      errorText = tr("Your account has reach the limit of how many Authentication Addresses it may have in submitted state. Please verify your currently submitted addresses before submitting further addresses");
```

Comment:
"has reached", and please wrap around this line - it's too long.

Reply:
done.